src/containers/SignupForm/index.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Button } from 'react-materialize';
+import PropTypes from 'prop-types';
+import * as M from 'materialize-css';
```

Comment:
Avoid naming variables as just a single letter. Make this more descriptive

Reply:
noted.